Content.Shared/Implants/SharedImplanterSystem.cs

```diff
@@ -46,17 +50,22 @@ private void OnEntInserted(EntityUid uid, ImplanterComponent component, EntInser
     //Set to draw mode if not implant only
     public void Implant(EntityUid user, EntityUid target, EntityUid implanter, ImplanterComponent component)
     {
-        if (!CanImplant(user, target, implanter, component, out var implant, out var implantComp))
+        if (!CanImplant(user, target, implanter, component, out var implant, out var implant2, out var implantComp, out var implantComp2))
             return;
 
         //If the target doesn't have the implanted component, add it.
         var implantedComp = EnsureComp<ImplantedComponent>(target);
         var implantContainer = implantedComp.ImplantContainer;
 
         component.ImplanterSlot.ContainerSlot?.Remove(implant.Value);
+        component.ImplanterSlot2.ContainerSlot?.Remove(implant2.Value);
```

Comment:
```suggestion
```

Remove all references to implant2.

Implanters are designed to only have a singular implant in them and no more.
If you really wanted to fit in a second implant you'd want to have a collection of implants rather than another slot. But again they're designed for a singular implant only.

Reply:
I removed all the code referencing the second implant slot